File: web3rpc/rpc-specs/code-samples/python/eth/block/getUncleCountByBlockHash.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

blockHash = "0xc9dbfbab67e9a0508bcb3f95ae408023668cef431b805592781a821926715b8a"

w3 = Web3(Web3.HTTPProvider(host))
eth_response = w3.eth.get_uncle_count_by_block_hash(blockHash)

print(eth_response)

File: web3rpc/rpc-specs/code-samples/javascript/eth/block/getUncleCountByBlockHash.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
let sdk = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
    const blockHash = '0xc9dbfbab67e9a0508bcb3f95ae408023668cef431b805592781a821926715b8a'
    
    sdk.eth.getUncleCountByBlockHash(blockHash, {}, (err, data, response) => {
        console.log(data);
    });
}
)()